### PR TITLE
allows fermi_energy to be zero

### PR DESCRIPTION
**Pull request by AndresOrtegaGuerrero**
This PR allows the widget to be use for vibroscopy plugin to set fermi to zero

### CODE REVIEW

File: src/aiidalab_qe/common/bandpdoswidget.py [hunk 1]
```diff
@@ -479,15 +479,15 @@ def handle_spin_polarization(fermi_energy, update_func, layout_type):
         # PDOS plot type
         if self.plot_type == "pdos":
             fermi_energy = self.fermi_energy.get(
-                "fermi_energy"
-            ) or self.fermi_energy.get("fermi_energy_up")
+                "fermi_energy", self.fermi_energy.get("fermi_energy_up")
```
Comment (mikibonacci) [inline src/aiidalab_qe/common/bandpdoswidget.py:482]: `self.fermi_energy.get("fermi_energy_up",0)` ? So we are sure that we get 0 fermi, in the worst case scenario

Comment (AndresOrtegaGuerrero) [reply src/aiidalab_qe/common/bandpdoswidget.py:482]: no , it should check in case there is one or the other , there is always one or the other

Comment (mikibonacci) [reply src/aiidalab_qe/common/bandpdoswidget.py:482]: Got it, thanks!

**Review by mikibonacci — APPROVED** [review]

**Comment by mikibonacci** [comment]
Hi @AndresOrtegaGuerrero LGTM, thanks!

**Pull request merged**